crates/uv/tests/it/python-downloads-metadata.json

```diff
@@ -0,0 +1,674 @@
+{
```

Comment:
Should this be here?

Reply:
Ah I see it's being used for the test case. Interesting 🤔 I'm worried about maintaining these. Do we need a subset? If so, should it be much smaller?

Reply:
I think we need a fixed subset to keep things stable; otherwise, the tests fall apart every time we sync the latest Python versions.

So, I’m making sure the subset includes a prerelease, a freethreaded variant, different patch versions of the same minor version, and different minor versions. This way, we’ve got a pretty thorough test case.

Reply:
Hm.

I'm not sure if I mind if the snapshots need updating when the available versions change.

My concern is that the snapshot doesn't capture the reality for most users, which makes it less valuable and more likely for us to accidentally ship regressions.